onnxruntime/core/platform/windows/env.cc

```diff
@@ -137,9 +158,21 @@ class WindowsThread : public EnvThread {
     }
     unsigned ret = 0;
     ORT_TRY {
+      // TODO: should I try to use SetThreadSelectedCpuSets?
+      if (p->affinity_mask.has_value()) {
+        auto rc = SetThreadAffinityMask(GetCurrentThread(), *p->affinity_mask);
+        if (!rc) {
+          const auto error_code = GetLastError();
+          LOGS_DEFAULT(ERROR) << "SetThreadAffinityMask failed for thread: " << GetCurrentThreadId()
+                              << " error code: " << error_code
+                              << " error msg: " << std::system_category().message(error_code)
+                              << " Specify the number of threads explicitly so the affinity is not set.";
```

Comment:
>" Sp [](http://example.com/codeflow?start=33&length=4)

nit: if previous error msg not end with a period, the sentence might look a bit weird. For example:
"error msg: not enough resources Specify the number of threads ...", further - "Specify the number of threads explicitly so the affinity is not set", this might change in near future. Suggest we doc this in the release pertain to 1.13, instead of putting it here.